test/testdata/packager/lsp_features/bart/bart.rb.document-symbols.exp

```diff
@@ -80,6 +55,31 @@
                     "children": []
                 }
             ]
+        }, {
+            "name": "CatchPhrase",
+            "detail": "",
+            "kind": 14,
+            "range": {
+                "start": {
+                    "line": 2,
+                    "character": 4
+                },
+                "end": {
+                    "line": 2,
+                    "character": 15
+                }
+            },
+            "selectionRange": {
+                "start": {
+                    "line": 2,
+                    "character": 4
+                },
+                "end": {
+                    "line": 2,
+                    "character": 15
+                }
+            },
+            "children": []
```

Comment:
Do you know why this moved around?

Reply:
Yup. Iteration order changed since there are different symbol tables now.

Reply:
This feature still isn't released; I couldn't tell you if VS Code actually respects the order of things here, or if it uses document positions to reorder them.